lib/ice_cube/validations/daily_interval.rb

```diff
@@ -4,7 +4,11 @@ module Validations::DailyInterval
 
     # Add a new interval validation
     def interval(interval)
-      @interval = normalized_interval(interval)
+      interval = normalized_interval(interval)
+      verify_alignment(interval, :wday, :interval) { |error| raise error }
```

Comment:
Why do we yield this error up instead of just `raise`ing it?

Reply:
This is so that it gives a more relevant trace, since it appears from the called `interval` public method interface, instead of raising from somewhere deeper down.